docs/buildingpex.rst

```diff
@@ -63,6 +64,22 @@ You can then import and manipulate modules like you would otherwise::
     >>> psutil.__path__
     ['/private/var/folders/4d/9tz0cd5n2n7947xs21gspsxc0000gp/T/tmpYuGpFW/.deps/psutil-2.0.0-py2.6-macosx-10.4-x86_64.egg/psutil']
 
+Specifying source directories
+-----------------------------
+
+Source directories are source distributions typically built with ``python setup.py sdist``.
+The ``-s`` option specifies the path to a source directory.::
+
+    $ python setup.py sdist
+    $ tar -xzvf dist/foo-x.y
+    $ pex -s foo-x.y
```

Comment:
This is equivalent to 'pex -s .'

I think it makes sense to skip the sdist part because it might confuse readers into thinking they need to first create an sdist, when pex does exactly this under the covers.  An alternative would be to describe either git cloning a repository with a setup.py or fetching a source distribution and then unpacking and running 'pex -s' on the target.